frontend/src/metabase/containers/CollectionItemsLoader.jsx

```diff
@@ -13,7 +13,14 @@ const CollectionItemsLoader = ({ collectionId, children, ...props }: Props) => (
     {({ object }) => (
       <Search.ListLoader
         {...props}
-        query={{ collection: collectionId }}
+        query={{
+          collection: collectionId,
+          pinned_state: "is_pinned",
+          sort_column: "name",
+          sort_direction: "asc",
+          models: "dashboard",
+          limit: 200,
```

Comment:
@flamber @mazameli can you please share how many folders we can show on the home page under "Our analytics"?
<img width="1388" alt="Screen Shot 2021-07-07 at 16 29 05" src="https://user-images.githubusercontent.com/14301985/124767405-7643fc00-df40-11eb-8967-8fb5773785d7.png">